tests/Doctrine/Tests/CouchDB/Functional/CouchDBClientTest.php

```diff
@@ -412,4 +412,100 @@ public function testGetRevisionDifference()
         }
         $this->assertEquals($revisionDifference, $revDiff);
     }
+
+    /**
+     * @depends testCreateBulkUpdater
+     */
+
+    public function testTransferChangedDocuments()
+    {
+        $client = $this->couchClient;
```

Comment:
Are these tests really using the multipart client? Where can I see that that is the client that's being used?